python-package/lightgbm/compat.py

```diff
@@ -146,6 +148,14 @@ class dask_Array:  # type: ignore
 
         pass
 
+    def dask_array_from_delayed(*args, **kwargs):  # type: ignore
+        """Mock function for dask.array.from_delayed()."""
+        pass
+
+    def dask_bag_from_delayed(*args, **kwargs):  # type: ignore
+        """Mock function for dask.bag.from_delayed()."""
+        pass
```

Comment:
Sorry, missed this moment in my previous review. Why dummy function and not simply `None` as for all other functions in this module?
https://github.com/microsoft/LightGBM/blob/e36cc9c1717665963a7fed09a5413a2bf5f0f50a/python-package/lightgbm/compat.py#L113-L122
https://github.com/microsoft/LightGBM/blob/e36cc9c1717665963a7fed09a5413a2bf5f0f50a/python-package/lightgbm/compat.py#L135-L137

Reply:
oh! No reason, I just didn't notice those. I'll update that here.

Reply:
updated in https://github.com/microsoft/LightGBM/pull/4378/commits/e4fed6f1300983993babe32111325ca98c448e5d